src/ws-notification/ws-notification.js

```diff
@@ -94,7 +94,7 @@ export class WSNotification extends Component {
    * @returns {void}
    */
   animateIn(notification, index) {
-    const list = this.refs.list;
```

Comment:
you should have fix this in another smaller PR. I guess you're fixing an issue here in the "free-dropdown" PR. We should still separate stuff no matter how small it is :)